velox/connectors/lakehouse/iceberg/ConnectorConfigBase.cpp

```diff
@@ -16,9 +16,7 @@
 
```

Comment:
Since it's already in the final leaf folder, there should not be XXXBase classes or files anymore. These Base classes shall be merged with the final classes, e.g. ConnectorConfigBase shall be merged into IcebergConfig